src/node-lib/getServerConfig.ts

```diff
@@ -193,13 +193,17 @@ const envVars = satisfies<Record<string, EnvVar>>()({
     default: null,
   },
   clerkPublishableKey: {
+    description:
+      "Note: this value must also be set in Netlify's environment vars since it is needed in middleware",
```

Comment:
This is annoying but the oak config .env files are not sourced in middleware (which runs in deno) and the only way to set them is to follow Netlify conventions and set them in their UI.